### PR TITLE
Currency converter for eyb spend form

### PR DESCRIPTION
**Pull request by stuart-madetech**
Currency converter for eyb spend form

_Tick or delete as appropriate:_

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/IOO-731
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: international_online_offer/forms.py [hunk 1]
```diff
@@ -117,33 +110,38 @@ class HiringForm(forms.Form):
...
 class SpendForm(forms.Form):
+
     spend = ChoiceField(
-        label='What is your planned spend for UK entry or expansion?',
-        help_text="""This is for the first three years of your project.
-        Choose an estimated amount from the list, or enter a specific amount""",
+        label='Select an estimate',
         required=True,
-        widget=forms.RadioSelect(attrs={'id': 'spend-select', 'onclick': 'handleSpendRadioClick(this);'}),
+        widget=forms.RadioSelect(attrs={'id': 'spend-select'}),
```
Comment (timothyPatterson) [inline international_online_offer/forms.py:117]: forms.RadioSelect resolves to an unordered list [see here](https://github.com/uktrade/great-cms/blob/develop/templates/great_components/form_widgets/multiple_input.html). Moving forward I think the idea is to use `GreatRadioSelect` there is an example PR [here](https://github.com/uktrade/great-cms/pull/2947).

File: international_online_offer/templates/eyb/triage/spend.html [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
+            {% include '../includes/triage_header.html' with back_url=back_url step_text=step_text question_text=question_text %}
+        </div>
         <div class="govuk-grid-row govuk-!-margin-bottom-3">
-            <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-grid-column-full">
+                <form method="get" action="#spend-container">
+                    {% csrf_token %}
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-one-third-from-desktop spend-currency-column">
+                            {% include '../includes/form_field.html' with field=spend_currency_from.spend_currency %}
```
Comment (timothyPatterson) [inline international_online_offer/templates/eyb/triage/spend.html:20]: Should this be `spend_currency_form`?

Comment (stuart-madetech) [reply international_online_offer/templates/eyb/triage/spend.html:20]: No its just custom padding styling for the one column only

**Review by timothyPatterson — APPROVED** [review]

File: international_online_offer/forms.py (hunk 1 above)
Comment (stuart-madetech) [reply international_online_offer/forms.py]: Updated to use :)

**Pull request merged**